pkgs/applications/graphics/ImageMagick/default.nix

```diff
@@ -11,7 +11,7 @@ let
     else throw "ImageMagick is not supported on this platform.";
 
   cfg = {
-    version = "6.9.3-9";
+    version = "6.9.5-2";
     sha256 = "0q19jgn1iv7zqrw8ibxp4z57iihrc9kyb09k2wnspcacs6vrvinf";
```

Comment:
IIRC I had left the mingw version old, because the patch required for it to build didn't apply on newer versions and they didn't compile by themselves.